### PR TITLE
Add reference and BAM/CRAM index to QualityYield.

### PR DESCRIPTION
**Pull request by jasonwalker80**

### CODE REVIEW

File: tasks/Qc.wdl [hunk 1]
```diff
@@ -18,16 +18,22 @@ version 1.0
 task CollectQualityYieldMetrics {
   input {
     File input_bam
+    File? input_bam_index
```
Comment (acoffman) [inline tasks/Qc.wdl]: Am I missing it or do we need need this passed in?

Comment (jasonwalker80) [reply tasks/Qc.wdl]: line 101 in WholeGenomeSingleSampleQc.wdl?

Comment (jasonwalker80) [reply tasks/Qc.wdl]: Or do you mean passed in to the Picard comamnd? Picard does not actually accept the BAM/CRAM index as an input. However, in order to localize data on the host, Cromwell has to be told it's an input so it retains the index with the actual BAM/CRAM.

Comment (acoffman) [reply tasks/Qc.wdl]: Got it, that makes sense!

**Pull request merged**